rust/scx_utils/src/gpu.rs

```diff
@@ -51,6 +51,9 @@ pub fn create_gpus() -> BTreeMap<usize, Vec<Gpu>> {
             };
 
             let cpu_mask = if let Ok(cpu_affinity) = nvidia_gpu.cpu_affinity(*NR_CPUS_POSSIBLE) {
```

Comment:
BTW, this should use `NR_CPU_IDS`. Possible CPUs are not guaranteed to be consecutive.